packages/@azure/servicebus/data-plane/test/stress/fixedNumberOfClientsMultipleQueues.ts

```diff
@@ -0,0 +1,62 @@
+import { ServiceBusClient, SendableMessageInfo } from "../../lib";
+
+const connectionString = "";
+
+const testDurationInMilliseconds = 60000 * 5 * 12 * 24 * 7; // 1 week
+
+const numOfClients = 10;
+
+let msgId = 1;
+
+let snapshotIntervalID: any;
+
+let isJobDone = false;
+
+async function main(): Promise<void> {
+  snapshotIntervalID = setInterval(snapshot, 5000); // Every 5 seconds
+  setTimeout(() => {
+    isJobDone = true;
+  }, testDurationInMilliseconds);
+
+  sendReceiveMessages();
+}
+
+async function sendReceiveMessages(): Promise<void> {
+  const ns = ServiceBusClient.createFromConnectionString(connectionString);
+
+  const clients = [];
+  for (let i = 0; i < numOfClients; i++) {
+    clients[i] = ns.createQueueClient(`t1-queue-new-${i + 1}`);
+  }
+
+  try {
+    while (!isJobDone) {
+      for (let i = 0; i < numOfClients; i++) {
+        const sender = clients[i].createSender();
```

Comment:
The second time around inside the while loop, we are attempting to call `createSender()` on the clients the second time. As part of https://github.com/Azure/azure-sdk-for-js/pull/1761 which was merged 2 days ago, this will now throw an error, `An open sender already exists on this QueueClient. Please close it and try again or use a new QueueClient instance`

Prior to https://github.com/Azure/azure-sdk-for-js/pull/1761 everytime we called `getSender`/`createSender`, the same sender was being returned

So, am guessing the intention here is to re-use the senders and receivers. 
In that case, have an array of senders and receivers outside the while loop just like you do for the clients.

Same for `createReceiver`